Dockerfile.ci

```diff
@@ -0,0 +1,32 @@
+FROM circleci/golang:1.11.5
+ENV PROTOC_VERSION 3.6.1
+ENV VAULT_VERSION 1.1.0
+ENV CONSUL_VERSION 1.4.0
+
+RUN sudo apt-get update \
+    && sudo apt-get -y upgrade \
+    && sudo apt-get install -y unzip \
+    && sudo rm -rf /var/lib/apt/lists/*
```

Comment:
What's this for? Just saving some space?

Reply:
Saving some space and general image hygiene